migrations/20210717161239_add-admin-session-tokens.up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE "admin_session_tokens"
+(
+    "user_id"    uuid      NOT NULL REFERENCES users (id),
```

Comment:
```suggestion
    "user_id"    uuid      NOT NULL REFERENCES users (id) ON DELETE CASCADE,
```

we don't need this session token row if such user id does not exist

Reply:
oke, i used to think we gonna never delete user, may be only set inactive/archive it)

Reply:
@Drevoed fixed